index.html

```diff
@@ -10,19 +10,34 @@
 </head>
 
 <body>
-    <header>
+    <header id="hello">
         <div class="logo"><a href="">Johnson</a> </div>
-        <nav> <a href=""><img class="image" src="images/Union.png" alt="dropdown"></a> </nav>
+        <nav>
+            <img id="hamburgerBtn" class="image" src="images/Union.png" alt="dropdown">
+        </nav>
         <nav class="primary-nav">
             <a href="">Portfolio</a>
             <a href="">About</a>
             <a href="">Contact</a>
             <a href=""><img class="msg-icon" src="images/msg-icon.png" alt="msg-icon"></a>
         </nav>
     </header>
+
     <main>
 
         <section class="headline">
+            <div class="menu-slider">
+                <div class="close">
+                    <img id="closeBtn" src="images/X-button.svg" alt="close button">
+                </div>
+                <div class="menu-nav-item">
+                    <ul class="menu-navigation">
+                        <li class="menu-item"><a id="seePortfolio" class="menu-links" href="#portfolio">Portfolio</a> </li>
+                        <li class="menu-item"><a id="aboutMe" class="menu-links" href="#About">About</a> </li>
+                        <li class="menu-item"><a id="contactMe" class="menu-links" href="#contact">contact</a> </li>
+                    </ul>
```

Comment:
- [ ] The border-bottom on the mobile menu items run through to the end of the right display. In the given design, there's an equal space on the left and right of the mobile menu. I will attach an image below for further clarification 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/173572434-b1420dca-a4dd-461a-a560-e75dc352e32d.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/173572536-eb6a88b9-0b69-4e8f-a923-38ed98dacfc4.png)

**Hint: While looking at your code, I realized that you have a `padding-left: 40px` on your `menu-navigation`. Doing the same on the right could be an easy fix for this issue.**